etc/profile-a-l/evolution.profile

```diff
@@ -40,7 +70,27 @@ novideo
 protocol unix,inet,inet6
 seccomp
 shell none
+tracelog
 
+# disable-mnt
+# Add "gpg,gpg2,gpg-agent,pinentry-curses,pinentry-emacs,pinentry-fltk,pinentry-gnome3,pinentry-gtk,pinentry-gtk2,pinentry-gtk-2,pinentry-qt,pinentry-qt4,pinentry-tty,pinentry-x2go,pinentry-kwallet" for gpg
```

Comment:
If private-bin doesn't support gpg by default (which really I think we should support) then shouldn't we block access to ~/.gnupg ?

Reply:
`private-bin` is disabled (see https://github.com/netblue30/firejail/pull/3607#discussion_r479185506), this means gpg is in $PATH.

Reply:
@rusty-snake 🤦 yep!